tests/integration/components/topics/list-item-test.js

```diff
@@ -0,0 +1,13 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('topics/list-item', 'Integration | Component | topics/list item', {
+  integration: true
+});
+
+test('displays the topic name', function(assert) {
+  let topic = { name: 'topic one' };
+  this.set('topic', topic);
+  this.render(hbs`{{topics/list-item topic=topic}}`);
+  assert.equal(this.$('.topic').text(), 'topic one\n');
```

Comment:
The "\n" is more about the topic name being wrapped in a block element, so the next element ends up in a new line.  That seems to get translated to a new line character when jquery asks for the text in the parent element.  

Probably should use `this.$('.topic').text().trim()` instead though.